controllers/spacebindingcleanup/spacebinding_cleanup_controller_test.go

```diff
@@ -12,13 +12,13 @@ import (
 	"github.com/codeready-toolchain/host-operator/pkg/cluster"
 	. "github.com/codeready-toolchain/host-operator/test"
 	sb "github.com/codeready-toolchain/host-operator/test/spacebinding"
-	spacebindingrequesttest "github.com/codeready-toolchain/host-operator/test/spacebindingrequest"
 	commoncluster "github.com/codeready-toolchain/toolchain-common/pkg/cluster"
 	commonconfig "github.com/codeready-toolchain/toolchain-common/pkg/configuration"
 	"github.com/codeready-toolchain/toolchain-common/pkg/test"
 	testconfig "github.com/codeready-toolchain/toolchain-common/pkg/test/config"
 	"github.com/codeready-toolchain/toolchain-common/pkg/test/masteruserrecord"
 	spacetest "github.com/codeready-toolchain/toolchain-common/pkg/test/space"
+	spacebindingrequesttestcommon "github.com/codeready-toolchain/toolchain-common/pkg/test/spacebindingrequest"
```

Comment:
maybe `spacebindingrequesttestcommon` can be shortened to be `sbrtestcommon`? given we already use short hands `sb` and `sbr` for space binding and space binding requests accordingly.

Reply:
Good point. Thanks !

Done in https://github.com/codeready-toolchain/host-operator/pull/970/commits/42943c251fca64c951bbf8097ae9bed72aaac7e3